microproxy/layer/application/http1.py

```diff
@@ -15,204 +13,73 @@ def __init__(self, context):
         super(Http1Layer, self).__init__()
         self.context = context
         self.src_conn = Connection(
-            self.context.src_stream, False, self, our_role=h11.SERVER)
+            h11.SERVER,
+            self.context.src_stream,
+            conn_type="src",
```

Comment:
conn_type is only for debug.
It's ok to removed it.
But if remove it, we are hard to which Connection receive a h11 event or send h11 event.